### PR TITLE
Modulesync 0.7.0

### PR DESCRIPTION
**Pull request by madAndroid**

### CODE REVIEW

**Comment by madAndroid** [comment]
Just one error with this: (as shown at: https://travis-ci.org/voxpupuli/puppet-jenkins_job_builder/jobs/135703824)
```
Running RuboCop...
Inspecting 6 files
..C...

Offenses:

spec/classes/jenkins_job_builder_spec.rb:198:99: C: Style/RegexpLiteral: Use %r around regular expression.  (https://github.com/bbatsov/ruby-style-guide#percent-r)
      it { expect { should contain_package('jenkins_job_builder') }.to raise_error(Puppet::Error, /Nexenta not supported/) }
                                                                                                  ^^^^^^^^^^^^^^^^^^^^^^^

6 files inspected, 1 offense detected
RuboCop failed!
```
@bastelfreak / @jyaworski - I'm not sure I agree with the style here: the more common usage for regex syntax is the `/ /` construct - should we look at changing this particular cop?

**Comment by bastelfreak** [comment]
We changed that to %r{} after a longer discussion on modulesync_config here: https://github.com/voxpupuli/modulesync_config/issues/168

I would like to stay with %r{}

**Comment by madAndroid** [comment]
@bastelfreak - yeah, not having to escape slashes is a good argument, I guess .. it's just a little surprising for those used to using the more well known syntax .. I'll amend the specs to use `%r{}` :)

**Comment by madAndroid** [comment]
@bastelfreak - sorted :)

File: .travis.yml [hunk 1]
```diff
@@ -35,11 +35,10 @@ deploy:
   provider: puppetforge
   user: puppet
   password:
-    secure: "A7lnigQt5QnJQfafRBPEQo1xSmEarUMZtw16SdBkQFSrMb/PJuihrIHwEUOzYK1lhXArxJQ5fB1WW1QYGaVCygewQNOgUje2dUER0DyuphWoJf1woun7p5VbZE1q0SV1U8wmDitViUe1iC7q9rFTaA3oMfVUxQaD4IsrCOTQqcE="
```
Comment (bastelfreak) [inline .travis.yml]: can you add that to .sync.yml and back to .travis.yml? otherwise we can't deploy through travis

Comment (madAndroid) [inline .travis.yml]: hmm.. looks like that's already in `.sync.yaml`? not sure why it would have been removed .. I'll have a look

Comment (madAndroid) [inline .travis.yml]: ah, hangon .. the difference is between `secure:` and `secret:` ?

Comment (bastelfreak) [inline .travis.yml]: ah, we broke that :D 
secure should be correct.

Comment (madAndroid) [inline .travis.yml]: whoops :D

**Comment by madAndroid** [comment]
@bastelfreak - hopefully that's the last of the changes - `.travis.yml` amended, and updated in `.sync.yml`

**Comment by bastelfreak** [comment]
awesomo \o/ thanks

**Pull request merged**